storage/database/dynamodb.go

```diff
@@ -58,32 +60,34 @@ const dynamoMaxRetry = 5
 const WorkerNum = 10
 const itemChanSize = WorkerNum * 2
 
-var overSizedDataPrefix = []byte("oversizeditem")
+var (
+	dynamoDBClient    *dynamodb.DynamoDB          // handles dynamoDB connections
+	dynamoOnceWorker  sync.Once                   // makes sure worker is created once
+	dynamoWriteCh     chan *batchWriteWorkerInput // use global write channel for shared worker
+	dynamoOpenedDBNum uint
+)
 
 type DynamoDBConfig struct {
 	TableName          string
 	Region             string // AWS region
 	Endpoint           string // Where DynamoDB reside
 	IsProvisioned      bool   // Billing mode
-	ReadCapacityUnits  int64  // read capacity when provsioned
-	WriteCapacityUnits int64  // write capacity when provsioned
+	ReadCapacityUnits  int64  // read capacity when provisioned
+	WriteCapacityUnits int64  // write capacity when provisioned
 }
 
 type batchWriteWorkerInput struct {
-	items []*dynamodb.WriteRequest
-	wg    *sync.WaitGroup
+	tableName string
+	items     []*dynamodb.WriteRequest
+	wg        *sync.WaitGroup
+	logger    log.Logger
```

Comment:
How about declaring a new logger inside of `createBatchWriteWorker` or a global logger and printing a log with `tableName`?

Reply:
I've changed to use global logger.